.travis-docker.sh

```diff
@@ -16,16 +16,17 @@ opam_version=${OPAM_VERSION:-$default_opam_version}
 base_remote_branch=${BASE_REMOTE_BRANCH:-$default_base_remote_branch}
 
 # create env file
-echo PACKAGE="$PACKAGE" > env.list
-echo EXTRA_REMOTES="$EXTRA_REMOTES" >> env.list
-echo PINS="$PINS" >> env.list
-echo INSTALL="$INSTALL" >> env.list
-echo DEPOPTS="$DEPOPTS" >> env.list
-echo TESTS="$TESTS" >> env.list
-echo REVDEPS="$REVDEPS" >> env.list
-echo EXTRA_DEPS="$EXTRA_DEPS" >> env.list
-echo PRE_INSTALL_HOOK="$PRE_INSTALL_HOOK" >> env.list
-echo POST_INSTALL_HOOK="$POST_INSTALL_HOOK" >> env.list
+rm -f env.list
+if [ -n "${PACKAGE+x}" ] ; then echo PACKAGE="$PACKAGE" >> env.list ; fi
```

Comment:
`${FOO+x}` expands to `x` if `FOO` has a value **or** if `FOO=` but _not_ if `FOO` isn't in the environment. So an explicit `PACKAGE=` from the .yml file will still be passed into the container.